frontend/src/lib/components/ActivityLog/activityLogLogic.tsx

```diff
@@ -90,7 +90,7 @@ export const activityLogLogic = kea<activityLogLogicType>({
                 (pageScope === ActivityScope.PERSON && hashParams['activeTab'] === 'history') ||
                 (pageScope === ActivityScope.FEATURE_FLAG && searchParams['tab'] === 'history') ||
                 (pageScope === ActivityScope.INSIGHT && searchParams['tab'] === 'history') ||
-                (pageScope === ActivityScope.PLUGIN && searchParams['tab'] === 'history')
+                pageScope === ActivityScope.PLUGIN
```

Comment:
note that this is hacky and the main thing that needs fixing in a follow up PR.

It's the reason I ended up putting this behind a flag, but not fixing this here as the PR is already bloated with refactors and this requires another one